spec/factories/qc_file_factories.rb

```diff
@@ -1,16 +1,29 @@
 # frozen_string_literal: true
 
 FactoryBot.define do
+  factory :qc_file, class: Sequencescape::Api::V2::QcFile do
+    skip_create
+
+    uuid
+
+    content_type { 'text/csv' }
+    contents { 'example,file,content' }
+    created_at { Time.new('2017-06-29T09:31:59.000+01:00') }
+    filename { 'file.csv' }
+    labware { build(:plate) }
+    size { 123 }
+  end
+
   # API V1 JSON description of QC file
-  factory :qc_file, class: Sequencescape::QcFile, traits: [:api_object] do
+  factory :v1_qc_file, class: Sequencescape::QcFile, traits: [:api_object] do
```

Comment:
Yeah I didn't want to create new factories with `v2` in the name when I knew the `v1` stuff was coming out.  This is a point of discussion because when we remove all the v1 factories, will we want the remaining ones to have tha name `v2_` or should they all just be named as the thing they are?  Katy said it could be good either way.  Perhaps if we put `v2_` in the name and a `v3_` comes along or new endpoints are added to v2, people will create factories without the numbering in.